users/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
OY, I'm pretty sure those shouldn't be empty. At the very least tests for views, models and forms should be implemented

Reply:
It appears @MartynaAnnaGottschling wanted this PR out as fast as possible to get overall feedback regarding her implementation, and thus omitting tests for now.
So no hard feelings Martyna! 😄

But yeah, it won't be merged until those tests are done anyway.